server/src/main/java/org/elasticsearch/rest/action/ingest/RestSimulateIngestAction.java

```diff
@@ -140,7 +140,7 @@ static BytesReference convertToBulkRequestXContentBytes(Map<String, Object> sour
      * simulate-style xcontent.
      * Non-private for unit testing
      */
-    static class SimulateIngestRestToXContentListener extends RestToXContentListener<BulkResponse> {
+    static class SimulateIngestRestToXContentListener extends RestBuilderListener<BulkResponse> {
```

Comment:
Why not make this one chunked too?

Reply:
Guess it's not really worth it, who's going to send something large enough so that it needs chunking to the simulate API? :D

Reply:
famous last words...